rskj-core/src/main/java/co/rsk/net/sync/DownloadingBodiesSyncState.java

```diff
@@ -88,7 +88,8 @@ public void newBody(BodyResponseMessage message, MessageChannel peer) {
 
         // we already checked that this message was expected
         BlockHeader header = pendingBodyResponses.remove(message.getId()).header;
-        Block block = Block.fromValidData(header, message.getTransactions(), message.getUncles());
+        Block block = new Block(header, message.getTransactions(), message.getUncles());
+        block.seal();
```

Comment:
I would try a function here that encapsulates block creation, something like the ones that were created for clone and modifiableClone

Reply:
That is needed for `unitrie` and is done in that branch. I think we should wait.